project-management/src/main/java/life/qbic/projectmanagement/domain/service/ProjectDomainService.java

```diff
@@ -70,7 +74,8 @@ public Result<Project, ResponseCode> registerProject(
   }
 
   public enum ResponseCode {
-    PROJECT_REGISTRATION_FAILED
+    PROJECT_REGISTRATION_FAILED,
+    PROJECT_ALREADY_EXISTS
```

Comment:
You probably want to add a message to message.properties

Reply:
As far as i understood it the message.properties only handle the ApplicationExceptions generated by the application services. For this i already used the implemented Errorcode: 
DUPLICATE_PROJECT_CODE=The project Code {0} already exists -> A valid project code must be unique